.github/workflows/release.yml

```diff
@@ -51,25 +63,35 @@ jobs:
         run: |
           make install
           make build VERSION=${{ steps.version.outputs.content }} BIN_DIR_WIN=./build
+      - name: Get gpu-post release
+        shell: bash
+        run: |
+          mkdir build_${{ runner.os }}
+          unzip libgpu-setup-*.zip -d build_${{ runner.os }}
+
+          #Download latest smrepl release
+          wget https://github.com/spacemeshos/smrepl/releases/latest/download/${{ runner.os }}.zip
```

Comment:
I think we should pin the version here rather than using `latest`, it's safer

Reply:
Fixed.